src/App/Pages/Settings/SettingsPage/SettingsPageViewModel.cs

```diff
@@ -161,7 +161,8 @@ public async Task TwoStepAsync()
                 AppResources.TwoStepLogin, AppResources.Yes, AppResources.Cancel);
             if(confirmed)
             {
-                _platformUtilsService.LaunchUri("https://help.bitwarden.com/article/setup-two-step-login/");
+                var twoFAURL = _cozyClientService.GetURLForApp("settings", fragment: "/profil");
```

Comment:
It should profile with an e no ? I see

```
<Route exact path="/profile" component={Profile} />
```

in Settings.

Reply:
Yep you're right thx. It works because requests are redirected to ... /profile 🤦